cocos/audio/include/AudioEngine.h

```diff
@@ -113,60 +113,77 @@ class EXPORT_DLL AudioEngine
      * @lua endToLua
      */
     static void end();
-    
-    /**  
+
+    /**
      * Gets the default profile of audio instances.
      *
      * @return The default profile of audio instances.
      */
     static AudioProfile* getDefaultProfile();
-    
-    /** 
+
+    /**
      * Play 2d sound.
      *
      * @param filePath The path of an audio file.
      * @param loop Whether audio instance loop or not.
      * @param volume Volume value (range from 0.0 to 1.0).
+     * @param pitch Pitch value (0.0f, any]
      * @param profile A profile for audio instance. When profile is not specified, default profile will be used.
      * @return An audio ID. It allows you to dynamically change the behavior of an audio instance on the fly.
      *
      * @see `AudioProfile`
      */
     static int play2d(const std::string& filePath, bool loop = false, float volume = 1.0f, const AudioProfile *profile = nullptr);
-    
-    /** 
+
+    /**
      * Sets whether an audio instance loop or not.
      *
      * @param audioID An audioID returned by the play2d function.
      * @param loop Whether audio instance loop or not.
      */
     static void setLoop(int audioID, bool loop);
 
-    /** 
+    /**
      * Checks whether an audio instance is loop.
      *
      * @param audioID An audioID returned by the play2d function.
      * @return Whether or not an audio instance is loop.
      */
     static bool isLoop(int audioID);
 
-    /** 
+    /**
      * Sets volume for an audio instance.
      *
      * @param audioID An audioID returned by the play2d function.
      * @param volume Volume value (range from 0.0 to 1.0).
      */
     static void setVolume(int audioID, float volume);
 
-    /** 
+    /**
+     * Sets pitch for an audio instance.
+     *
+     * @param audioID An audioID returned by the play2d function.
+     * @param pitch Pitch value, 0.5 = half pitch, 2.0 = double pitch, etc default = 1.0.
```

Comment:
`default = 1.0` does it mean if we don't set pitch, the default is 1.0 set by the player itself as i don't see there is any codes to set the pitch.